extensions/voice/org.eclipse.smarthome.voice.mactts/src/main/java/org/eclipse/smarthome/voice/mactts/internal/MacTTSService.java

```diff
@@ -89,17 +90,21 @@ public AudioStream synthesize(String text, Voice voice, AudioFormat requestedFor
      */
     private final Set<Voice> initVoices() {
         Set<Voice> voices = new HashSet<Voice>();
+        InputStreamReader inputStreamReader = null;
+        BufferedReader bufferedReader = null;
         try {
             Process process = Runtime.getRuntime().exec("say -v ?");
-            InputStreamReader inputStreamReader = new InputStreamReader(process.getInputStream());
-            BufferedReader bufferedReader = new BufferedReader(inputStreamReader);
+            inputStreamReader = new InputStreamReader(process.getInputStream());

```

Comment:
> I would have to put one try-catch block in addition to the try-with-recources block.

That's correct. And I don't see anything against the encapsulation of try blocks.
But if you prefer using the finally block here and you are sure that there is no corner case we need to close the inputStreamReader, too (don't know if the BufferedReader constructor could throw a runtime excpection)...